Assets/MRTK/SDK/StandardAssets/Prefabs/ArticulatedHandMesh.prefab

```diff
@@ -57,7 +57,7 @@ MeshRenderer:
   m_RenderingLayerMask: 4294967295
   m_RendererPriority: 0
   m_Materials:
-  - {fileID: 2100000, guid: 45082b98567b3d44ca3bdbe39f46041a, type: 2}
+  - {fileID: 2100000, guid: 8898ca407d928454d876a616ba1be32e, type: 2}
```

Comment:
@cre8ivepark @davidkline-ms it looks like this PR updated the default hand mesh material to the mesh pulse material:

![image](https://user-images.githubusercontent.com/3580640/115316831-05562a80-a12f-11eb-9ede-f4ad888440a9.png)

but, since there doesn't seem to be a corresponding change to add the `PulseShaderHandMeshHandler`, which means the hand mesh doesn't show up anymore by default :( should this be reverted?